src/core/usubscription.rs

```diff
@@ -42,6 +42,14 @@ impl SubscriberInfo {
     }
 }
 
+impl SubscriptionResponse {
+    pub fn is_state(&self, state: State) -> bool {
```

Comment:
can you add some documentation and a doc test?

Reply:
done